src/unit/test_intset.c

```diff
@@ -0,0 +1,218 @@
+#include <sys/time.h>
+#include <time.h>
+
+#include "../intset.c"
+#include "test_help.h"
+
+#if 0
```

Comment:
I know this block was in existing code - but any reason to keep it?

Reply:
Sure, just to clarify this is just the draft I had. It's not really all that cleaned up.